tests/EnumTest.php

```diff
@@ -150,6 +179,9 @@ public function testIsValidKey()
     public function testSearch()
     {
         $this->assertEquals('FOO', EnumFixture::search('foo'));
-        $this->assertNotEquals('FOO', EnumFixture::isValidKey('baz'));
+        /**
+         * @see https://github.com/myclabs/php-enum/issues/9
+         */
+        $this->assertEquals(EnumFixture::PROBLEMATIC_NUMBER, EnumFixture::search(1));
```

Comment:
Here we search for `1` and we check it returns `PROBLEMATIC_NUMBER` whose value is `0`, is that normal?